### PR TITLE
Testfile and option changes

### PR DESCRIPTION
**Pull request by leepatrickworrall**

### CODE REVIEW

**Comment by leepatrickworrall** [comment]
Big confuse on why the tests were running, and now aren't running.

**Review by chellygel — COMMENTED** [review]
will need to finish review but this is a start

File: ketchup/main.py [hunk 1]
```diff
@@ -1,121 +1,228 @@
...
+                    continue
+                else:
+                    break
+
+            episode_duration_len_in_mins = int(input(
+                "What is the length of the episodes within the show you are "
+                "wanting to watch?: "
+            ))
+
+            episode_duration_mins = episode_duration_len_in_mins
```
[the remark this answers is not in the document]
Comment (leepatrickworrall) [reply ketchup/main.py:71]: Yeah there was no point of me having the default value being set on line 83 again. Good catch on that, thank you. I think it was for an older iteration.

File: ketchup/main.py [hunk 2]
```diff
@@ -1,121 +1,228 @@
...
+    print("Hello and welcome to AnimeKetchup "
+          "for all your episode catch-up needs!")
     while True:
         try:
-            user_date = input(prompt)
-            user_date = DDP.get_date_data(user_date).date_obj
+            user_option = int(input(
+                "Please choose an option:\n"
+                "1) Watch specified amount of episodes without any limits.\n"
+                "2) Watch an allotted amount of episodes per day.\n"
```
[the remark this answers is not in the document]
Comment (leepatrickworrall) [reply ketchup/main.py:42]: Removed, and added as a "TODO" before the run() function.

**Pull request closed without merging**